pulsar-client-cpp/include/pulsar/CompressionType.h

```diff
@@ -20,11 +20,7 @@
 #define PULSAR_COMPRESSIONTYPE_H_
 #pragma GCC visibility push(default)
 namespace pulsar {
-enum CompressionType {
-    CompressionNone = 0,
-    CompressionLZ4 = 1,
-    CompressionZLib = 2
-};
+enum CompressionType { CompressionNone = 0, CompressionLZ4 = 1, CompressionZLib = 2 };
```

Comment:
Is there a way to tell `clang-format` to keep enums in separate lines?

Reply:
BreakBeforeBraces
BraceWrapping

Reply:
@merlimat @maskit We can keep enums in separate lines by this, but the new line inserted between the definition and the brace.
https://github.com/apache/incubator-pulsar/pull/1080/files#diff-f5034129a6e7700b144e19b41534c1afR25
https://github.com/apache/incubator-pulsar/pull/1080/files#diff-2327e49613e902fe01fad24b083bc888R23